### PR TITLE
Add serathius to SIG instrumentation

### PR DESCRIPTION
**Pull request by serathius**
/assign @brancz

### CODE REVIEW

File: config/kubernetes-incubator/org.yaml [hunk 1]
```diff
@@ -147,6 +147,7 @@ members:
 - sarahnovotny
 - sbezverk
 - sebgoa
+- serathius
```
Comment (mrbobbytables) [inline config/kubernetes-incubator/org.yaml]: Hey there @serathius, a couple things:
- Org membership changes should be their own separate commit and have the commit message be something like: `Add serathius to kubernetes-incubator org`. 
- Is joining incubator intended to help with migration to @kubernetes-sigs?  With [incubator's deprecation](https://github.com/kubernetes/community/issues/1922#issuecomment-516974448) we're not adding new members to @kubernetes-incubator unless it's absolutely essential. cc @brancz

Comment (serathius) [reply config/kubernetes-incubator/org.yaml]: Thanks for review, I created separate PR for kubernetes-incubator https://github.com/kubernetes/org/pull/1379

**Comment by serathius** [comment]
@brancz can you approve?

**Comment by cblecker** [comment]
/approve
(as @brancz already indicated he's good with this)

**Comment by brancz** [comment]
Sorry, yes! :slightly_smiling_face: :tada: 

/approve

**Pull request merged**